packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -635,28 +635,24 @@ export default class Agent extends EventEmitter<{|
     appendComponentStack,
     breakOnConsoleErrors,
     showInlineWarningsAndErrors,
+    hideDoubleLogsInStrictMode,
+    __REACT_DEVTOOLS_SHOW_INLINE_WARNINGS_AND_ERRORS__,
```

Comment:
This looks like a mistaken copy/paste?

I think you meant:
```suggestion
    hideDoubleLogsInStrictMode,
```